### PR TITLE
fix: version on tx history + createAnother on prepareDelegateAuthority

### PR DESCRIPTION
**Pull request by andreabadesso**
### Acceptance Criteria
- We should return the `version` attribute on transactions history
- `prepareDelegateAuthority` on the wallet-service facade should expect `createAnother` instead of `createAnotherAuthority` param

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

**Review by pedroferreira1 — APPROVED** [review]

File: src/new/wallet.js [hunk 1]
```diff
@@ -1154,6 +1154,7 @@ class HathorWallet extends EventEmitter {
           tokenUid,
           balance: tokenTxBalance,
           voided: tx.is_voided,
+          version: tx.version,
```
[the remark this answers is not in the document]
Comment (andreabadesso) [reply src/new/wallet.js]: Done in 23a082f05e1b23b1c2499818ba762b2a8903fe13

File: src/wallet/wallet.ts [hunk 2]
```diff
@@ -1518,12 +1518,12 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
...
     type optionsType = {
       anotherAuthorityAddress: string | null,
-      createAnotherAuthority: boolean,
+      createAnother: boolean,
       pinCode: string | null,
     };
     const newOptions: optionsType = Object.assign({
       anotherAuthorityAddress: null,
-      createAnotherAuthority: true,
+      createAnother: true,
```
[the remark this answers is not in the document]
Comment (andreabadesso) [reply src/wallet/wallet.ts]: Good idea! Refactored in f3eec45f5acc19987be350f5901a14e9b03cc93f

File: src/new/wallet.js [hunk 3]
```diff
@@ -1122,6 +1122,17 @@ class HathorWallet extends EventEmitter {
     return history;
   }
 
+  static getTokenHistoryObject(tx, tokenUid, tokenTxBalance) {
```
[the remark this answers is not in the document]
Comment (andreabadesso) [reply src/new/wallet.js:1125]: Done in f3eec45f5acc19987be350f5901a14e9b03cc93f

**Review by r4mmer — APPROVED** [review]

File: src/wallet/types.ts [hunk 4]
```diff
@@ -259,6 +260,16 @@ export interface IStopWalletParams {
   cleanStorage?: boolean;
 };
 
+export interface DelegateAuthorityOptions {
+  anotherAuthorityAddress: string | null;
+  createAnother: boolean | null;
```
Comment (pedroferreira1) [inline src/wallet/types.ts:265]: From the previous code, this can't be `null`, right?

**Review by pedroferreira1 — CHANGES_REQUESTED** [review]

File: src/new/wallet.js [hunk 5]
```diff
@@ -1135,6 +1135,17 @@ class HathorWallet extends EventEmitter {
...
```
[the remark this answers is not in the document]
Comment (andreabadesso) [reply src/new/wallet.js:1150]: Done in f94fc1cd609c5a908add71759b84dc70bb4700d8

File: src/wallet/types.ts (hunk 4 above)
Comment (andreabadesso) [reply src/wallet/types.ts]: You are right, not sure why I set it to `null`, fixed in 5c6285111de7755d091e7be1884dccef6aca3975

**Review by pedroferreira1 — APPROVED** [review]

**Review by pedroferreira1 — APPROVED** [review]

**Pull request merged**